gcc/rust/expand/rust-attribute-visitor.cc

```diff
@@ -3472,4 +3392,18 @@ AttrVisitor::visit (AST::BareFunctionType &type)
 
   // no where clause, apparently
 }
+void
+AttrVisitor::maybe_expand_expr (std::unique_ptr<AST::Expr> &expr)
+{
+  auto fragment = expander.take_expanded_fragment (*this);
+  if (fragment.should_expand ())
+    expr = fragment.take_expression_fragment ();
+}
+void
```

Comment:
```suggestion
}

void
```
EXTREME nitpicking haha